articles/azure-functions/durable/durable-functions-entities.md

```diff
@@ -19,28 +19,52 @@ Entity functions define operations for reading and updating small pieces of stat
 > [!NOTE]
 > Entity functions and related functionality is only available in Durable Functions 2.0 and above. Entity functions are currently in public preview.
 
-## Entity identity
+## General concepts
```

Comment:
We may need to move this PR into the private repo so that we can validate whether any links were broken when renaming subsections. There are also substantial content changes which means we'll need to run the automated tooling to ensure the article follows the Azure documentation guidelines appropriately.

Reply:
I don't think I removed any anchors (at least not intentionally). 

Yes, let's get together so I can understand the tooling better.